internals/secrethub/migrate.go

```diff
@@ -272,9 +272,6 @@ func (cmd *MigratePlanCommand) addDirToPlan(client secrethub.ClientInterface, pa
 		}
 		return nil
 	}
-	if err != nil {
```

Comment:
This err is now unhandled.

We should still keep

```go
if err != nil {
    return err
}
```

for the cases where the error is not forbidden or not found.

Reply:
Adjusted properly.